common/web/lm-worker/src/main/correction/execution-timer.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * This inner class is designed to help the algorithm detect its active execution time.
```

Comment:
Is this comment still correct? This inner class...

Reply:
It's adjusted in the next PR (#11758).